### PR TITLE
Fix for API docs generator

### PR DESCRIPTION
**Pull request by pcppcp**
- enabled napoleon extension to parse google docs properly
- removed sphinx-api call - sphinx can run it by itself
- fixed some docstrings

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ SPHINXBUILD      = sphinx-build
 SPHINXAPIDOC     = sphinx-apidoc
 PAPER            =
 BUILDDIR         = ./_build
-RAIDENMODULEPATH = ..
```
Comment (LefterisJP) [inline docs/Makefile]: doesn't this depend on where you run the `make` command from?

File: docs/conf.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 # - Add Raiden path to the path for sphinx to import
 import os
 import sys
-sys.path.insert(0, os.path.dirname(os.path.abspath('.')))
```
Comment (LefterisJP) [inline docs/conf.py]: same here?

Comment (pcppcp) [inline docs/conf.py]: It doesn't matter, you can run any of the

`cd docs && make html
make -C docs html
make docs`

the result is the same, with -C option make will change its working dir.

Comment (LefterisJP) [inline docs/conf.py]: Understood. Thank you.

**Pull request merged**